### DISCUSSION TITLE
How do I load a external <script> libaray into Nue

### DISCUSSION THREAD

**Discussion by kyoukhana** [nuejs/nue#336] [General]
For instance,  I was wondering how can I load https://alpinejs.dev/ into Nue.

**Comment by logistor** [comment]
You can add head tag into layout.html

```
<head>
     <script src="//unpkg.com/alpinejs" defer></script>
</head>
```

**Reply by nobkd** [reply]
Ref: https://nuejs.org/docs/reactivity.html#google-analytics

Also, you may (or may not, (I'm not sure at the moment, would have to check it)) need to add `<slot for="head" />` ([ref](https://nuejs.org/docs/custom-layouts.html#root-layout)) to allow automatic imports for e.g. css files, setting title, etc.

**Reply by logistor** [reply]
No need for slot tag. I have tested it.

**Comment by kyoukhana** [comment]
Just as a test I guess you need to do this

```
  <script client>
    console.log('Test');
  </script>
```

They Key would be adding client as well in the <script> tag
